libs/server/Storage/Session/ObjectStore/SetOps.cs

```diff
@@ -735,7 +735,7 @@ public unsafe GarnetStatus SetIsMember<TObjectContext>(ArgSlice key, ArgSlice[]
             {
                 type = GarnetObjectType.Set,
                 SetOp = SetOperation.SMISMEMBER,
-            }, ref parseState, 0);
```

Comment:
SetIsMember does not need to construct a new local parseState:
 
            var parseState = new SessionParseState();
            parseState.InitializeWithArguments(members);